mobizel/markdown-docs-bundle/0.1/docs/index.md

```diff
@@ -1,3 +1,5 @@
 # Documentation
```

Comment:
Why do I, as a user, need this file?

Reply:
This is the homepage of your documentation.
If you don't have it, you have a not found. With this first markdown file you can check everything is setup correctly and I think it's important for the DX part.

Reply:
Okey. I thought it was just for documentation. 

Thank you for clarifying.